lint/setup.cfg

```diff
@@ -0,0 +1,29 @@
+# This file is part of sv-witnesses repository: https://github.com/sosy-lab/sv-witnesses
+#
+# SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[metadata]
+name = sv-witness-lint
+version = attr: witnesslint.__version__
+author = Dirk Beyer
+description = A linter for the sv-witness format
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/sosy-lab/sv-witnesses/tree/master/lint
+project_urls =
+    Bug Tracker = https://github.com/sosy-lab/sv-witnesses/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: Apache Software License
+    Intended Audience :: Science/Research
+    Environment :: Console
+license = Apache-2.0
+license_file = LICENSES/Apache-2.0.txt
+
```

Comment:
Wouldn't this file at least need `[options.entry_points]`? Did you try out the generated package?

(But this depends on whether we select an alternative build tool.)

Reply:
Right, the package can be imported and used without `[options.entry_points]`, but cannot be used from the commandline. However, even if I add
```
[options.entry_points]
console_scripts =
    witnesslinter = witnesslint.linter:main
```
I only get an error message about a missing `witnesslint.__main__` module. Of course I can add this, but then the `[options.entry_points]` is unnecessary. What am I missing here?

Reply:
Not sure. It is similar to what we do in BenchExec. We don't use `__main__` there because we have more than one entry point in the same package. If that works here, why not? (Other packaging tools might have their own best practices on how to define entry points, though.)

Reply:
Then I shall leave it as is for now and look for an appropriate solution once we decided on a packaging tool to use.

Reply:
Using the poetry equivalent
```
[tool.poetry.scripts]
witnesslint = "witnesslint.linter:main"
```
it works even without adding `witnesslint.__main__`.